free-2.md

```diff
@@ -1,51 +1,65 @@
 
 
 ## The user and a language
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
+This project would serve people working in consulting, finance or accounting who want to automate or speed up the process of data retrieval and population of Excel spreadsheets. 
 
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+People working in these fields often have to parse large amount of information, often in enormous PDF files. These documents like the 10-K contain data in tables that have to be manually copied and pasted into Excel spreadsheets with relationships and equations between cells systematically entered. This is often quite tedious and repetitive work - suggesting an opportunity for a DSL. With this DSL, certain simple equations could be simplified to commands, and strung together with the fluency of a programming language. Ideally, the use of this DSL would free up time and resources for people to devote on less mind-numbing activities. 
+
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+It would be able to automate some of this tedious work and improve productivity. 
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+It could potentially be a very useful DSL. I came across this idea after conversations with peers working in economic consulting and my personal experience in Econ classes like Intro to Accounting. 
 
 ### Domain
 _Describe the project's domain in five words._
 
+Automated Excel spreadsheet population
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+At the most basic level, a user could interact with the language on command-line. Although this is not very ideal as it may not be very user-friendly for non-programmers, but would suffice as a minimum-viable product. This would be the right way to interact with the problem domain because creating a web GUI would take away time from development of the MVP. 
 
 ### Operation (semantics)
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+An initial call to the program could isolate all the tables in the PDF and paragraphs containing keywords entered by the user. Certain frequently used equations can be functions in our language. Further terminal commands could recreate each table in Excel and replicate some of the relationships between the cells that are laborious to enter on Excel. As the behavior in each table usually varies, we would probably want to allow for each table to be analyzed and populated individually onto Excel. 
```

Comment:
Does this mean your program would be parsing PDFs?  I can't tell if it's just the user looking at the PDF or if the program will be interacting with and parsing the PDF.  I don't know exactly what that entails, but given the PDF standard, this could be quite a task!

What exactly are these "frequently used equations" you are referring to?